src/include/processor/warning_context.h

```diff
@@ -19,20 +40,41 @@ struct PopulatedCSVError {
 
 struct WarningInfo {
     uint64_t queryID;
-    processor::PopulatedCSVError warning;
+    PopulatedCSVError warning;
 
-    WarningInfo(processor::PopulatedCSVError warning, uint64_t queryID)
+    WarningInfo(PopulatedCSVError warning, uint64_t queryID)
         : queryID(queryID), warning(std::move(warning)) {}
 };
 
-struct WarningContext {
-    std::vector<WarningInfo> warnings;
-    main::ClientConfig* clientConfig;
-    std::mutex mtx;
+using populate_func_t = std::function<PopulatedCSVError(const SharedWarning*)>;
```

Comment:
Will remove/merge with `populate_error_func_t` once `WarningContext` is cleaned up

Reply:
removed `populate_error_func_t`